types/pixelmatch/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for pixelmatch 4.0
+// Project: https://github.com/mapbox/pixelmatch#readme
+// Definitions by: Oleg Repin <https://github.com/iamolegga>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare function Pixelmatch(
+  /** Image data of the first image to compare. Note: image dimensions must be equal. */
+  img1: Buffer | Uint8Array,
+  /** Image data of the second image to compare. Note: image dimensions must be equal. */
+  img2: Buffer | Uint8Array,
+  /** Image data to write the diff to, or null if don't need a diff image. */
+  output: Buffer | Uint8Array | null,
+  /** Width of the images. Note that all three images need to have the same dimensions. */
+  width: number,
+  /** Height of the images. Note that all three images need to have the same dimensions. */
+  height: number,
+  /** Options. */
+  options: Options | undefined
```

Comment:
It's cleaner to have this be `options?: Options` if the intent is that options may not be provided.

Reply:
Yes, sure!